### PR TITLE
Add Kibana dashboard link to Commercial Admin page

### PR DESCRIPTION
**Pull request by JonNorman**
## What does this change?

Sometimes commercial feeds fail - no, they do - and we only ever find out from angry emails or by accident. Now we can just take a look at [a dashboard](http://kibana.gu-web.net/app/kibana#/dashboard/Commercial-Feeds?_g=%28refreshInterval:%28display:Off,pause:!f,value:0%29,time:%28from:now-4h,mode:quick,to:now%29%29&_a=%28filters:!%28%29,options:%28darkTheme:!t%29,panels:!%28%28col:1,id:'Failing-Commercial-Feeds-%28Fetch%29',panelIndex:2,row:1,size_x:6,size_y:4,type:visualization%29,%28col:7,id:'Failing-Commercial-Feeds-%28Parse%29',panelIndex:3,row:1,size_x:6,size_y:4,type:visualization%29,%28col:1,columns:!%28commercial-feed,commercial-feed-action,message%29,id:'Commercial-Feeds-%28Failing%29',panelIndex:4,row:5,size_x:12,size_y:3,sort:!%28'@timestamp',desc%29,type:search%29%29,query:%28query_string:%28analyze_wildcard:!t,query:'*'%29%29,title:'Commercial%20Feeds',uiState:%28P-2:%28spy:%28mode:%28fill:!f,name:!n%29%29%29%29%29) (ideally, stuck up on one of these screens around the office).

Parse failures are more important to notice than fetch failures as a failed parse almost certainly indicates something wrong on our end; fetches fail more frequently but cause less concern as external feeds are not especially reliable. We don't want to set alarms for these but having the ability to look in real time (+7 days historically) will make it easier to manage the feeds if and when there is an actual issue that is within our gift to fix.

**note: currently Kibana is tokenising the soulmates feed name - e.g. soulmates/brighton - which has led to a large "soulmates" curve of failing fetches; this has been fixed by sending an indexing template to Elastic Search that will keep the feed name intact, but it takes time to come through the system.**
## What is the value of this and can you measure success?

Fewer angry emails about feeds having been down for ages.

### CODE REVIEW

File: admin/app/views/commercial/commercialMenu.scala.html [hunk 1]
```diff
@@ -112,6 +112,9 @@
                     <ul>
                         <li><a href="https://github.com/guardian/commercial-tools">Commercial tools repo</a></li>
                         <li><a href="https://dfp-playground.appspot.com/59739193">DFP API playground</a></li>
+                        <li><a href="http://kibana.gu-web.net/app/kibana#/dashboard/Commercial-Feeds?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-4h,mode:quick,to:now))&_a=(filters:!(),options:(darkTheme:!t),panels:!((col:1,id:'Failing-Commercial-Feeds-(Fetch)',panelIndex:2,row:1,size_x:6,size_y:4,type:visualization),(col:7,id:'Failing-Commercial-Feeds-(Parse)',panelIndex:3,row:1,size_x:6,size_y:4,type:visualization),(col:1,columns:!(commercial-feed,commercial-feed-action,message),id:'Commercial-Feeds-(Failing)',panelIndex:4,row:5,size_x:12,size_y:3,sort:!('@timestamp',desc),type:search)),query:(query_string:(analyze_wildcard:!t,query:'*')),title:'Commercial%20Feeds',uiState:(P-2:(spy:(mode:(fill:!f,name:!n)))))">
```
Comment (regiskuckaertz) [inline admin/app/views/commercial/commercialMenu.scala.html]: That's what I call a link 😆  Minor fix: `&` usually indicates a parameter entity, that's why in HTML you have to write `&amp;`

**Comment by regiskuckaertz** [comment]
You might need to encode the URL parameters, but otherwise 👍

File: admin/app/views/commercial/commercialMenu.scala.html (hunk 1 above)
Comment (TBonnin) [inline admin/app/views/commercial/commercialMenu.scala.html]: You can have kibana generate a short link :smile:

Comment (JonNorman) [inline admin/app/views/commercial/commercialMenu.scala.html]: But what if I want to customise the chart using the URL parameters?!

Comment (TBonnin) [inline admin/app/views/commercial/commercialMenu.scala.html]: I am not sure I understand what you are trying to do?

Comment (JonNorman) [inline admin/app/views/commercial/commercialMenu.scala.html]: Sorry that was just a joke - I've updated it to use the short URL 😁

Comment (TBonnin) [inline admin/app/views/commercial/commercialMenu.scala.html]: 😆 😆 😆

**Pull request merged**